mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -1421,6 +1652,10 @@ def basic_coupled_ns_heat_operator(
         *,
         time=0.,
         interface_noslip=True,
+        interface_radiation=False,
+        wall_emissivity=None,
+        sigma=None,
```

Comment:
Should we default it to its actual value 5.67e-8?

Reply:
I worry about specifying default values for constants, because it invites the possibility of omitting them by accident, leading to silently incorrect results if the value is supposed to be something other than the default. Better to produce an error I think.